pkg/operator/controller/status_test.go

```diff
@@ -25,49 +24,37 @@ func TestComputeStatusConditions(t *testing.T) {
 		inputs      testInputs
 		outputs     testOutputs
 	}{
-		{"no namespace", testInputs{false, 0, 0, 0}, testOutputs{true, false, true}},
-		{"no ingresses, no routers", testInputs{true, 0, 0, 0}, testOutputs{false, false, true}},
-		{"scaling up", testInputs{true, 1, 0, 0}, testOutputs{false, true, false}},
-		{"scaling down", testInputs{true, 0, 1, 0}, testOutputs{false, true, true}},
-		{"0/2 ingresses available", testInputs{true, 2, 0, 2}, testOutputs{false, false, false}},
-		{"1/2 ingresses available", testInputs{true, 2, 1, 1}, testOutputs{false, false, false}},
-		{"2/2 ingresses available", testInputs{true, 2, 2, 0}, testOutputs{false, false, true}},
+		{"no namespace", testInputs{false, 0, 0}, testOutputs{true, false, true}},
+		{"no clusteroperator, no ingresscontroller deployments", testInputs{true, 0, 0}, testOutputs{false, false, true}},
```

Comment:
"no clusteroperator" is not a test input.  Looks like this is simply "no ingresscontrollers".